### PR TITLE
[MISC] Lung nodule detection

### PR DESCRIPTION
**Pull request by Rakshith2597**
This PR contains codes for the paper [Lung Segmentation and Nodule Detection in Computed Tomography Scan using a Convolutional Neural Network Trained Adversarially using Turing Test Loss](https://ieeexplore.ieee.org/abstract/document/9175649). 

I am attaching a screenshot of pytest output obtained on our server for reference
miriad_lung_nodule

This is part of the project MIRIAD: Many Incarnations of Screening of Radiology for High Throughput Disease Screening via Multiple Instance Reinforcement Learning with Adversarial Deep Neural Networks, sponsored by INTEL TECHNOLOGY INDIA PVT. LTD.

Principal Investigators:

Dr Debdoot Sheet (PI), Dr Nirmalya Ghosh (Co-PI)
Department of Electrical Engineering
Indian Institute of Technology Kharagpur

Dr Ramanathan Sethuraman (Co-PI)
Intel Technology India Pvt. Ltd.

### CODE REVIEW

**Review by sungmanc — APPROVED** [review]
Not mandatory, How about putting doc-string and type-hint for this PR?
